File: infrastructure/src/main/java/com/marvelcomics/brito/infrastructure/utils/AlertDialogUtils.java
package com.marvelcomics.brito.infrastructure.utils;

import android.app.AlertDialog;
import android.content.Context;

public class AlertDialogUtils {

    public static void showSimpleDialog(String title, String message, Context context) {
        new AlertDialog.Builder(context)
                .setTitle(title)
                .setMessage(message)
                .setCancelable(false)
                .setNeutralButton("Ok", (dialog, which) -> dialog.dismiss())
                .show();
    }
}

File: infrastructure/src/main/java/com/marvelcomics/brito/infrastructure/utils/AlertDialogUtils.kt
package com.marvelcomics.brito.infrastructure.utils

import android.content.Context
import android.content.DialogInterface
import androidx.appcompat.app.AlertDialog

object AlertDialogUtils {
    fun showSimpleDialog(
        title: String?,
        message: String?,
        context: Context?
    ) {
        AlertDialog.Builder(context!!)
            .setTitle(title)
            .setMessage(message)
            .setCancelable(false)
            .setNeutralButton(
                "Ok"
            ) { dialog: DialogInterface, _: Int -> dialog.dismiss() }
            .show()
    }
}
